<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tutorial</class>
 <widget class="QWidget" name="Tutorial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>650</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>141</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/burger.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="slidesLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>511</width>
     <height>361</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>370</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>370</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="exampleButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>84</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Play!</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
